src/js/components/CheckBox/CheckBox.js

```diff
@@ -50,6 +50,7 @@ const CheckBox = forwardRef(
       onMouseOut,
       onMouseOver,
       pad,
+      plainProp = false,
```

Comment:
```suggestion
      plain = false,
```

Any reason not to just refer to as plain?

Reply:
yeah in the `formfield` code we have 
```
     if (
            child &&
            child.type &&
            grommetInputNames.indexOf(child.type.displayName) !== -1 &&
            child.props.plain === undefined &&
            child.props.focusIndicator === undefined
          ) {
            return cloneElement(child, {
              plain: true,
              focusIndicator: false,
            });
          }
```
which sets `plain: true` since `checkbox`  would fall under this if statement... so I wasn't sure what else to name it